dashboards/grafana-dashboard-kruize-metrics.configmap.yaml

```diff
@@ -2162,37 +1997,19 @@ data:
           },
           "targets": [
             {
-              "datasource": {
-                "type": "prometheus",
-                "uid": "${DS_CRCP01UE1-PROMETHEUS}"
-              },
               "editorMode": "code",
-              "expr": "max(max_over_time(kruizeDB_max{method=\"loadExperimentByName\",status=\"success\"}[1m]))",
-              "hide": false,
-              "legendFormat": "Max perAPIcall in last 1m",
+              "expr": "aws_rds_free_storage_space_average{job=~\"cloudwatch-exporter.*\",dbinstance_identifier=~\"(kruize-prod|kruize-stage)\"} / 1024 / 1024",
```

Comment:
For now, it is fine but can we add additional templating var that allow choosing DB instances? 
Based on selection you can provide value in query.